src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -865,14 +865,14 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <Import Project="$(RazorDesignTimeTargets)" />
 
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(_RazorUpToDateReloadFileTypesAllowWorkaround)' != 'false'">
```

Comment:
Used in unit testing to verify things work with the design time targets once we've removed the workaround